README.md

```diff
@@ -12,24 +12,116 @@ employing different filters, like deviceId, number of entries to be listed and s
 http://host:port/device/{device_id}/history?lastN={lastN}&attr={attr}&dateFrom={dateFrom}&dateTo={dateTo}
 ```
 
+### Dependencies
+- [falcon](https://falconframework.org/)
+- [gunicorn](https://gunicorn.org/)
+- [gevent](http://www.gevent.org/)
+- [python-dateutil](https://pypi.org/project/python-dateutil/1.4/)
+
+
+### Configuration
+
+In order to properly start History, the following variables must be set. All
+values are the default ones, and they need to be modified to the respective services' address.
+
+Key                         | Purpose                                                      | Default Value
+----------------------------|--------------------------------------------------------------|----------------
+HISTORY_DB_ADDRESS          |History database's address                                    |"mongodb"
+HISTORY_DB_PORT             |History database's port                                       |27017
+HISTORY_DB_REPLICA_SET      |History database's replica set address                        |None
+HISTORY_DB_DATA_EXPIRATION  |Seconds before removing an entry in MongoDB                   |604800
+MONGO_SHARD                 |Activate the use of sharding or not                           |False
+AUTH_URL                    |Auth url address                                              |"http://auth:5000"
+
+### How to install
+
+#### Standalone setup
+
+As this service is written in **Python 2.7**, it is recommended that all packages are
```

Comment:
Just as a reminder: how about test history with Python 3.X and then set it as default? Python 2.7 will be deprecated next year (I think)

Reply:
Yes, but we're planning to upgrade it in the near future.